dspace-api/src/main/java/org/dspace/app/suggestion/oaire/DateScorer.java

```diff
@@ -0,0 +1,200 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.suggestion.oaire;
+
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+import java.util.GregorianCalendar;
+import java.util.List;
+
+import org.dspace.app.suggestion.SuggestionEvidence;
+import org.dspace.app.suggestion.SuggestionUtils;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.service.ItemService;
+import org.dspace.external.model.ExternalDataObject;
+import org.dspace.util.MultiFormatDateParser;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Implementation of {@see org.dspace.app.suggestion.oaire.EvidenceScorer} which evaluate ImportRecords
+ * based on the distance from a date extracted from the ResearcherProfile (birthday / graduation date)
+ * 
+ * @author Andrea Bollini (andrea.bollini at 4science dot it)
+ *
+ */
+public class DateScorer implements EvidenceScorer {
+
+    private String birthDateMetadata;
+
+    private String educationDateMetadata;
+
+    private String minDateMetadata;
+
+    private String maxDateMetadata;
+
+    private int birthDateDelta = 20;
+    private int birthDateRange = 50;
```

Comment:
Delta and Range are parameters for the calculation of an index, measuring the probability of the Author in repository to be the actual author of the suggested item.

Reply:
@frabacche : Could you please add a description into the code itself?  The numbers "20" and "50" don't make sense otherwise.  We should describe whether these numbers reference the year? or some other portion of the date?

Reply:
added